layers/+lang/lua/funcs.el

```diff
@@ -33,19 +42,34 @@
 
 
 ;; LSP Lua
-(defun spacemacs//lua-setup-lsp-emmy ()
-  "Setup LSP Lua."
+(defun spacemacs//lua-setup-lsp-emmy-lua ()
+  "Setup emmy Lua."
   (when lua-lsp-emmy-java-path
     (setq lsp-clients-emmy-lua-java-path lua-lsp-emmy-java-path))
   (when lua-lsp-emmy-jar-path
     (setq lsp-clients-emmy-lua-jar-path (expand-file-name lua-lsp-emmy-jar-path)))
   (setq lsp-enable-file-watchers lua-lsp-emmy-enable-file-watchers)
+  (set (make-local-variable 'lsp-disabled-clients) '(lua-language-server lsp-lua-lsp))
```

Comment:
This should be replaced by `setq-local`:
```elisp
(setq-local lsp-disabled-clients '(lua-language-server lsp-lua-lsp))
```

Reply:
resolved in new submition.